ansible/roles/ocp4-workload-debugging-workshop/tasks/post_workload.yml

```diff
@@ -169,11 +169,58 @@
         msg: "[argo cd] no deployments found"
       when: argocd_verify_dc is not defined
 
-    - name: "[argo cd] Checking deployments"
-      fail:
-        msg: "[argocd] Some/all deployments are not ready"
-      when: item.replicas is not defined or item.readyReplicas is not defined  or item.replicas != item.readyReplicas
-      loop: "{{ argocd_verify_dc|json_query('resources[*].status') }}"
+- name: Wait for the argocd-redis to be ready
+  k8s:
+    api_version: v1
```

Comment:
Those are read actions, right?
Why not use `k8s_info` ?

Reply:
I have to check what k8s_info does. I'm not ansible expert. I did it with what I know.

Reply:
That should definitely be k8s_info. k8s creates (or deletes) things. k8s_info queries them. Probably a copy/paste error from elsewhere.

Reply:
I created another pull request with k8s_info here https://github.com/redhat-cop/agnosticd/pull/4464